lib/index.js

```diff
@@ -5,43 +5,64 @@ var util       = require('util');
 var _          = require('lodash');
 var assert     = require('assert-plus');
 
+var helpers    = require('./helpers');
+var HttpError  = require('./baseClasses/HttpError');
 var RestError  = require('./baseClasses/RestError');
 var httpErrors = require('./httpErrors');
 var restErrors = require('./restErrors');
-var helpers    = require('./helpers');
 
 
+/**
+ * create an error object from an http status code.
+ * first arg is status code, all subsequent args
+ * passed on to the constructor. only works for regular
+ * HttpErrors, not RestErrors.
+ * @public
+ * @function makeErrFromCode
+ * @param    {Number} statusCode the http status code
+ * @returns  {Error}             an error instance
+ */
+function makeErrFromCode(statusCode) {
+    // assert!
+    assert.number(statusCode, 'statusCode');
+    assert.equal(statusCode >= 400, true);
+
+    var args = _.rest(_.toArray(arguments));
+    var name = helpers.errNameFromCode(statusCode);
+
+    var ErrCtor = httpErrors[name];
+
+    // assert constructor was found
+    assert.func(ErrCtor);
+
+    // pass every other arg down to constructor
+    return helpers.makeInstance(ErrCtor, args);
+}
+
 
 /**
  * create RestError subclasses for users. takes a string, creates a
  * constructor for them. magicks, again.
  * @public
  * @function makeError
- * @param    {String} name   the name of the error class to create
- * @param    {Number} [code] optional status code
- * @returns  {Object}
+ * @param    {String}   name       the name of the error class to create
+ * @param    {Number}   statusCode optional status code
+ * @returns  {Function}            a constructor function
  */
-function makeError(name, code) {
+function make(name, statusCode) {
```

Comment:
A little bit of an overloaded name. Perhaps something a little more descriptive?